corehq/apps/app_manager/suite_xml/xml_models.py

```diff
@@ -603,12 +603,20 @@ def get_all_xpaths(self):
                 for datum in getattr(frame, 'datums', []):
                     result.add(datum.value)
 
+        def _get_graph_config_xpaths(configuration):
+            result = set()
+            for config in configuration.configs:
+                result.add(config.xpath_function)
+            return result
+
         for field in self.fields:
             if field.template.form == 'graph':
                 s = etree.tostring(field.template.node)
                 template = load_xmlobject_from_string(s, xmlclass=GraphTemplate)
+                result = result.union(_get_graph_config_xpaths(template.graph.configuration))
```

Comment:
I like `result.update(...)` better; more analogous to dict.update/list.extend. (It's also trivially more efficient by not creating a new set, though that doesn't really matter in this case.)